app/controllers/items_controller.rb

```diff
@@ -34,7 +34,11 @@ def show
       @current_hold_count = current_member.active_holds.active_hold_count_for_item(@item).to_i
     end
 
-    ahoy.track "Item viewed", item_id: @item.id
+    # the `request.referer != request.url` condition prevents logging the view after
+    # placing a hold redirects back to the item page
+    if request.referer != request.url
```

Comment:
We could write a wrapper of some kind (it would be a great use for Redis, but we don't have it in the stack) that could limit the tracking dynamically according to a rule like "only track page views once every n minutes". But I think this is a good place to start.